.env.production

```diff
@@ -5,3 +5,4 @@ REACT_APP_GOOGLE_ANALYTICS_ID="G-KDP9B6W4H8"
 REACT_APP_FIREBASE_KEY="AIzaSyBcZWwTcTJHj_R6ipZcrJkXdq05PuX0Rs0"
 REACT_APP_AWS_API_ENDPOINT="https://api.uniswap.org/v1/graphql"
 THE_GRAPH_SCHEMA_ENDPOINT="https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3"
+REACT_APP_SENTRY_DISABLED=true
```

Comment:
Why is sentry turned off?

Reply:
We're expecting a lot of bug reports at the moment and running out of quota, temporarily disabled on product request until to not block the allowance for other teams before their rollout